Sources/PreviewGallery/PreviewGallery.swift

```diff
@@ -26,25 +26,37 @@ struct PreviewCellView: View {
   }
 }
 
+/// A SwiftUI View that displays a gallery of previews organized by modules.
+///
+/// `PreviewGallery` presents a list of modules, each containing its respective previews.
+/// If no previews are found, it displays a message indicating so.
```

Comment:
It would be good to include in the documentation things like examples of how to use it, like maybe the example that's currently in the README

Reply:
I also think it needs to be shown in a `NavigationStack` - might be worth double checking that